app/views/layouts/_footer.html.haml

```diff
@@ -2,28 +2,8 @@
   .container
     .row
       .col-md-4#about-growstuff
-        %ul
-          %li= link_to t('about'), "http://wiki.growstuff.org/index.php/About%20Growstuff"
-          %li= link_to t('our_values'), "http://wiki.growstuff.org/index.php/Values"
-          %li= link_to t('open_source'), "https://github.com/Growstuff/growstuff"
-          %li= link_to t('growstuff_team'), "http://wiki.growstuff.org/index.php/Team"
-          %li= link_to t('get_involved'), "http://wiki.growstuff.org/index.php/Get_involved"
+        != cms_snippet_content(:footer1)
```

Comment:
This is probably not a big deal (and I don't know anything about Comfortable Mexican Sofa), but would it make sense for these to have more meaningful names other than `:footer1`, `:footer2`, and `:footer3`? Maybe `:about`, `:legal`, and `:contact`? Or even `:left_column`, `:middle_column`, and `:right_column`?